src/app/accounts/register/register.component.html

```diff
@@ -10,23 +10,51 @@ <h2 class="tui-form__header">Register</h2>
                     <div class="tui-form__row">
                         <tui-input aria-label="Email" formControlName="email">Email</tui-input>
                         <tui-field-error formControlName="email"></tui-field-error>
-                        <div class="tui-form__field-note">We'll never share your email with anyone else.</div>
+                        <div class="tui-form__field-note">We'll never share your email with anyone
+                            else.
+                        </div>
                     </div>
                     <div class="tui-form__row">
-                        <tui-input-password aria-label="Password" formControlName="password">Password
+                        <tui-input aria-label="User Name" formControlName="nickname">
+                            User Name
```

Comment:
Username is 1 word i think

Reply:
I think this should be called nickname here anyways